sygnal/apnspushkin.py

```diff
@@ -79,9 +79,18 @@ class ApnsPushkin(ConcurrencyLimitedPushkin):
     Relays notifications to the Apple Push Notification Service.
     """
 
-    # Errors for which the token should be rejected
-    TOKEN_ERROR_REASON = "Unregistered"
-    TOKEN_ERROR_CODE = 410
+    # Errors for which the token should be rejected and not reused
+    # See https://developer.apple.com/documentation/usernotifications/setting_up_a_remote_notification_server/handling_notification_responses_from_apns  # noqa: E501
+    # for the full list of possible errors.
+    TOKEN_ERRORS = {
+        (400, "BadDeviceToken"),  # Invalid token
```

Comment:
This would mean that the client gave us a bogus token?

Reply:
Yep